serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/utils/apicurio/FileDescriptorUtils.java

```diff
@@ -855,6 +856,23 @@ private static MessageElement toMessage(FileDescriptorProto file, DescriptorProt
             ExtensionsElement extensionsElement = new ExtensionsElement(DEFAULT_LOCATION, "", values);
             extensions.add(extensionsElement);
         }
+        Map<String, List<FieldDescriptorProto>> extendsMap =
+                descriptor
+                        .getExtensionList()
+                        .stream()
+                        .collect(Collectors.groupingBy(FieldDescriptorProto::getExtendee));
+        for (Map.Entry<String, List<FieldDescriptorProto>> entry : extendsMap.entrySet()) {
+            ImmutableList.Builder<FieldElement> extendFields = ImmutableList.builder();
+
+            for (FieldDescriptorProto fd : entry.getValue()) {
+                FieldElement extendField = toField(file, fd, false);
+                extendFields.add(extendField);
+
+            }
+
+            ExtendElement extendElement = new ExtendElement(DEFAULT_LOCATION, entry.getKey(), "", extendFields.build());
+            extendElements.add(extendElement);
+        }
```

Comment:
What's going one here?

Reply:
They added a new field `extends` in the class `MessageElement`.

I noticed that in 4.3.0, they added `groups` and you handled it with an empty list. At first, I wanted to do that but I suspected that it was wrong so I had to dig in protobuf and `wire-schema` (what a pain 😭) to understand what it was supposed to be and I came up with that piece of code:

It's a piece of code that aims to handle extended fields